bears/gherkin/GherkinLintBear.py

```diff
@@ -0,0 +1,133 @@
+import json
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+from coalib.results.Result import Result
+
+_setting_map = {True: 'on',
+                False: 'off'}
+
+
+@linter(executable='gherkin-lint',
+        use_stderr=True,
+        use_stdout=False,
+        global_bear=True)
+class GherkinLintBear:
+    """
+    Use Gherkin to run linting on feature files
+    """
+
+    LANGUAGES = {'Gherkin'}
+    REQUIREMENTS = {NpmRequirement('gherkin-lint', '2')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala_devel@googlegroups.com'}
+    LICENSE = 'AGPL_3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+
+    def process_output(self, output, filename, file):
+        stderr = json.loads(output)
+        for fileErr in stderr:
+            filePath = fileErr['filePath']
+            for err in fileErr['errors']:
+                yield Result.from_values(
+                    origin='{} ({})'.format(
+                        self.__class__.__name__,
+                        err['rule']),
+                    message=err['message'],
+                    line=int(err['line']),
+                    file=filePath)
+
+    @staticmethod
+    def generate_config(filename, file,
+                        indentation: bool = False,
+                        name_length: bool = False,
+                        new_line_at_eof: bool = False,
+                        no_dupe_feature_names: bool = False,
+                        no_dupe_scenario_names: bool = False,
+                        no_duplicate_tags: bool = False,
+                        no_empty_file: bool = False,
+                        no_files_without_scenarios: bool = False,
+                        no_homogenous_tags: bool = False,
+                        no_multiple_empty_lines: bool = False,
```

Comment:
this would be an interesting bear; most languages have rules around how many consecutive blank lines are permitted.  Python's rules is 1 or 2 depending on context.

Reply:
IMO these `no_foo` should be `foo` with the default inverted